notebook/base/handlers.py

```diff
@@ -849,7 +850,7 @@ class APIVersionHandler(APIHandler):
 
     def get(self):
         # not authenticated, so give as few info as possible
-        self.finish(json.dumps({"version":notebook.__version__}))
+        self.finish(json.dumps({"version": notebook.__version__, "module": JUPYTER_NOTEBOOK_TAG}))
```

Comment:
There's no need to include the tag.  Having a successful response from `/api` and a check of the version is sufficient.  Please revert the changes in this file.

Reply:
actually, "/api" response includes a "version" field is general design;
we MUST choose a feature with higher significance, JUPYTER_NOTEBOOK_TAG;

Reply:
> actually, "/api" response includes a "version" field is general design;

That's a fair point. Returning a "version" key from an `/api` endpoint might be fairly common. 

Is there a reason we wouldn't default to hitting the `/api/status` endpoint? There is some rich, Jupyter-specific information there...

Reply:
The reason we're avoiding adding keys to the response model is that changes to the REST API affect many other applications (nteract, jupyterlab, voila, etc.). We should only make these kinds of changes when absolutely necessary. We shouldn't add a key just for this command.